components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,72 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+.sui-CoverBasic {
+  max-height: 676px;
+  position: relative;
+  width: 100%;
+
+  &.clickable {
+    cursor: hand;
+  }
+
+  &-gradient {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+  }
+
+  &-gradient:after {
+    background: -moz-linear-gradient(top, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.65) 100%); /* FF3.6+ */
```

Comment:
Autoprefixer will add automatically the prefixes you need for your supported browsers, so no need to add all here.